.yamato/yamato-ci.yml

```diff
@@ -0,0 +1,67 @@
+linux:
+  name: "Linux"
+  agent:
+    type: Unity::VM
+    image: cds-ops/cds-ubuntu-18.04-base:stable
+    flavor: b1.small
+  commands:
+    - git submodule init
+    - git submodule update
+    - mono bee.exe build::tundra2::master
+    - mono bee.exe run::tundra2-unittest::master
+  artifacts:
+    artifacts:
+      paths:
+        - "build/**/*"
+mac:
+  name: "Mac"
+  agent:
+    type: Unity::VM::osx
+    image: buildfarm/mac:stable
+    flavor: m1.mac
+  commands:
+    - git submodule init
+    - git submodule update
+    - mono bee.exe build::tundra2::master
+    - mono bee.exe run::tundra2-unittest::master
+  artifacts:
+    artifacts:
+      paths:
+        - "build/**/*"
+windows:
+  name: "Windows"
+  agent:
+    type: Unity::VM
+    image: cds-ops/win10-base:stable
+    flavor: b1.small
+  commands:
+    - git submodule init
+    - git submodule update
+    - bee.exe build::tundra2::master
+    - bee.exe run::tundra2-unittest::master
+  artifacts:
+    artifacts:
+      paths:
+        - "build/**/*"
+builds-zip:
+  name: "zPackage"
+  agent:
+    type: Unity::VM
+    image: cds-ops/cds-ubuntu-18.04-base:stable
+    flavor: b1.small
+  commands:
+    - git submodule init
+    - git submodule update
+    - rm -r .git unittest/googletest/.git
```

Comment:
`git export` maybe better?

Reply:
not sure how to do that with yamato, can you? or you are thinking we should export into some other directory and create the zip from that?

Reply:
Yeah, I was thinking the latter. You'd just `git export` to some other directory, then that directory is the artifact - it makes it cleaner for any other kind of assembling of 'packaged output.'